packages/experimental/Menu/src/MenuTrigger/MenuTrigger.tsx

```diff
@@ -22,7 +22,7 @@ export const MenuTrigger = stagedComponent((props: MenuTriggerProps) => {
     // state in the inner render so we can access the child component and its props.
     const child = childrenArray[0];
     const revisedState = getRevisedState(menuTrigger, child.props);
-    const revised = React.cloneElement(child, revisedState);
+    const revised = React.cloneElement(child, revisedState.props);
```

Comment:
One of my PRs might fix this too, so it's possible there'll be a conflict here in the future.

Reply:
I can wait for your PR's to merge and undo this bit then

Reply:
Latest merged PR addresses this issue.